src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise(function(resolve) {
+  logo.addEventListener('click', () => {
+    document.body.innerHTML += `
+    <div class="message">
+      Promise was resolved!
+    </div>
+    `;
+
+    resolve();
+  });
+});
+
+const promise2 = new Promise(function(resolve, reject) {
+  resolve(null);
+
+  setTimeout(() => {
+    document.body.innerHTML += `
+    <div class="error-message">
+      Promise was rejected!
+    </div>
```

Comment:
better not to use `innerHTML +=`
use `textContent`, `append, prepend` or `insertAdjacentHTML`